shared/actions/config/index.js

```diff
@@ -1,210 +1,310 @@
 // @flow
 import logger from '../../logger'
-import * as KBFSGen from '../kbfs-gen'
-import * as FsGen from '../fs-gen'
-import * as LoginGen from '../login-gen'
 import * as ConfigGen from '../config-gen'
-import * as TeamsGen from '../teams-gen'
-import * as Constants from '../../constants/config'
-import * as GregorCreators from '../gregor'
-import * as NotificationsGen from '../notifications-gen'
+import * as ChatGen from '../chat2-gen'
```

Comment:
marking this so its not skipped, likely best to look at this on its own vs a diff